tests/test_mlip_calculators.py

```diff
@@ -103,9 +105,12 @@ def test_invalid_device(architecture):
     [
         ("alignn", "cpu", {}),
         ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024"}),
-        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024/best_model.pt"}),
+        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024" / "best_model.pt"}),
```

Comment:
Looking at it, I have a feeling
```
MODEL_PATH / "v5.27.2024"
```
wants to become a module level constant.

Reply:
done